specification/reservations/resource-manager/Microsoft.Capacity/preview/2019-04-01/reservations.json

```diff
@@ -357,13 +357,20 @@
             "$ref" : "./examples/GetReservationDetails.json"
           }
         },
-        "parameters" : [ {
-          "$ref" : "#/parameters/ReservationIdParameter"
-        }, {
-          "$ref" : "#/parameters/ReservationOrderIdParameter"
-        }, {
-          "$ref" : "#/parameters/ApiVersionParameter"
-        } ],
+        "parameters": [
+            {
+                "$ref": "#/parameters/ReservationIdParameter"
+            },
+            {
+                "$ref": "#/parameters/ReservationOrderIdParameter"
+            },
+            {
+                "$ref": "#/parameters/ApiVersionParameter"
+            },
+            {
+                "$ref" : "#/parameters/AppendRenewPropertiesParameter"
```

Comment:
Please provide examples about this part (maybe other newly added properties as well)

Reply:
Provided.